### PR TITLE
Introduction updates

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

File: topics/introduction/slides/index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 - **Web-based** platform for computational biomedical research
   - Developed at Penn State, Johns Hopkins and G. Washington universities with substantial outside contributions
   - **Open source** under [Academic Free License](https://opensource.org/licenses/AFL-3.0)
-- More than 4,400 [citations](https://www.citeulike.org/group/16008/)
+- More than 4,800 [citations](http://www.citeulike.org/group/16008/)
```
Comment (nsoranzo) [inline topics/introduction/slides/index.html]: SSL certificate of citeulike expired on 8 June 2015.

File: topics/introduction/slides/index.html [hunk 2]
```diff
@@ -184,6 +186,14 @@
 
 ![Workflow interface](../images/RNA-Seq_workflow_editing.png)
 
+???
```
Comment (willdurand) [inline topics/introduction/slides/index.html]: Is that intentional?

Comment (nsoranzo) [reply topics/introduction/slides/index.html]: Yes, it's the syntax to start the speaker notes.

Comment (willdurand) [reply topics/introduction/slides/index.html]: oh ok, did not know that.

**Pull request merged**